test/metabase/test_runner/assert_exprs.clj

```diff
@@ -118,10 +118,10 @@
      :diffs    [[actual [only-in-expected only-in-actual]]]}))
 
 (defmethod t/assert-expr 'partial=
-  [message [_ expected & actuals]]
-  `(do ~@(for [actual actuals]
-           `(t/do-report
-             (partial=-report ~message ~expected ~actual)))))
+  [message [_ expected actual]]
+  {:pre [(some? expected) (some? actual)]}
```

Comment:
Previously this supported any number of args, e.g.

```clj
(is (partial= x y z))
```

which was basically the same as doing

```clj
(is (partial= x y))
(is (partial= x z))
```

After the changes here anything after the first two args (`z`) is just silently. That's not really much better than silently doing nothing with only one arg. I'm cool with only allowing exactly two args but we have to error if someone passes more than two, especially since it was previously totally fine.

This will also fail if you did

```clj
(is (partial= nil x))
```

since you're asserting `some?`. `partial= nil` is probably a dumb thing to be doing , but I don't think it's something that needs to be disallowed since the test will still _run_.

```suggestion
  [message [_ expected actual :as form]]
  {:pre [(= (count form) 3]}
```

I'd do it this way instead. count the `partial=` form itself. Note that `form` will be something like `(partial= x y)` so we want the count to be exactly `3`